integrations/argocd/scripts/argocd-terrascan-remote-scan.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/sh
+#
+#    Copyright (C) 2021 Accurics, Inc.
+#
+#	Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#		http://www.apache.org/licenses/LICENSE-2.0
+#
+#	Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+###
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+TERRASCAN_SERVER="https://${SERVICE_NAME}"
+IAC=${IAC_TYPE:-"k8s"}
+IAC_VERSION=${IAC_VERSION:-"v1"}
+CLOUD_PROVIDER=${CLOUD_PROVIDER:-"all"}
+REMOTE_TYPE=${REMOTE_TYPE:-"git"}
+
+SCAN_URL="${TERRASCAN_SERVER}/v1/${IAC}/${IAC_VERSION}/${CLOUD_PROVIDER}/remote/dir/scan"
+
```

Comment:
Can we please verify the whether the mandatory env variables (SERVICE_NAME, REMOTE_URL) have been correctly set? 

If not correctly set, we can error out before making calls to terrascan. If set we can print a sensible log message with mandatory env variables. 

These things just help us in better debugging